Tests/LibWeb/Text/input/Editing/execCommand-selectAll.html

```diff
@@ -33,5 +33,13 @@
         // Perform selectAll
         document.execCommand('selectAll');
         reportSelection();
+
+        // Make sure selectAll checks for the existence of a Selection object
+        println(`document.queryCommandEnabled('selectAll'): ${document.queryCommandEnabled('selectAll')}`);
+        println(`(new Document).queryCommandEnabled('selectAll'): ${(new Document).queryCommandEnabled('selectAll')}`);
+
+        // Should not crash
+        (new Document).execCommand('selectAll');
```

Comment:
Other browsers behave slightly differently to us here, since they throw an `InvalidStateError` when calling `execCommand` on an XML document.  Using `document.implementation.createHTMLDocument().execCommand("selectAll")` instead would show that we behave the same as other browsers in this case.

Reply:
Ah gotcha. I focused on the crash report but it's definitely better to include an actual HTMLDocument.